games/minimal/mods/default/init.lua

```diff
@@ -730,8 +730,10 @@ end
 
 -- Register nodes
 
+local S = core.get_translator("test")
+
 minetest.register_node("default:stone", {
-	description = "Stone",
+	description = S("Stone @1", minetest.colorize("red", "arg")),
```

Comment:
Is `minetest.colorize("red", "arg")` just here for demonstrating the PR? I assume we won't need to add this to every description in MTG?